vagrant/provisioning/arkcase-ce-facts.yml

```diff
@@ -303,6 +303,10 @@ ssl_version: "TLSv1.2"
 # ssl_cipher only for backward compatibility
 ssl_cipher: "{{ ssl_version }}"
 
+# Mongodb folder location
+mongodb_data_folder: "{{ root_folder }}/data/mongo"
+mongodb_log_folder: "{{ root_folder }}/log/mongodb"
```

Comment:
We don't need these facts here, since we are installing mongo only with arkcase-dev-microservices-facts.yml. Or, if they are used as documentation of sorts, then add these to arkcase-dev-facts.yml as well.